### DISCUSSION TITLE
CSV example import issue

### DISCUSSION THREAD

**Discussion by romaderi12** [vesoft-inc/nebula#4298] [General]
Hi

I tried to import the example with nebula importer and the sample script file provided at https://docs.nebula-graph.io/3.1.0/nebula-importer/config-without-header/ . Script throw the following error :+1: 
2022/06/03 13:37:29 yaml: line 94: did not find expected key

I tried to customize the script with my own data and encounter the same issue.
What's wrong ?

Regards,

**Comment by xiajingchun** [comment]
It seems the format of your yaml file is incorrect. Attach it here, and I can help to take a look.
